tests/e2e/helper.ts

```diff
@@ -14,3 +14,25 @@ export async function reloadWait(page: Page, element: Locator) {
     await page.reload();
     await expect(element).toBeVisible();
 }
+
+/**
+ * Gets the first and last name of the currently logged-in user.
+ *
+ * This function assumes that the user is logged in and the user menu is visible.
+ * It opens the user menu, retrieves the user's name, and closes the menu again.
+ *
+ * **Note**: This function is only necessary because the user name is changed in other tests and they interfere with each other.
+ *
+ * @param page - the current page
+ * @returns the first and last name of the user
+ */
+export async function getCurrentUser(page: Page): Promise<{ firstName: string; lastName: string }> {
+    await page.getByRole("button", { name: /[A-Z]{2}/ }).click();
```

Comment:
```suggestion
    await page.getByRole("button", { name: /^[A-Z]{2}$/ }).click();
```

If there are any two capital letters in the project which are on a button, they would also match, now they need to be at the start and end of a line making it more rare to match.
-> `home-page-model.ts`

Reply:
Good catch